src/utils/request.ts

```diff
@@ -0,0 +1,49 @@
+import axios, { AxiosRequestConfig } from 'axios';
```

Comment:
How much more verbose would it be to use `https` (potentially with our own wrapper) vs `axios`?

Reply:
Just removed Axios as dependency using node-fetch instead